src/main/java/games/strategy/debug/ConsoleHandler.java

```diff
@@ -33,6 +33,7 @@
   private final Supplier<PrintStream> errSupplier;
   private final Supplier<PrintStream> outSupplier;
 
+  @SuppressWarnings("unused")
```

Comment:
Just noting for other Eclipse users that this will show up as a warning: "Unnecessary `@SuppressWarnings("unused")`".  This happens at least in Neon.3 (I need to upgrade to Oxygen soon :smile:).  AFAIK, the Eclipse static analyzer does not attempt to look for unused public members.

Reply:
Not the same message, but an info message: 'At least one of the problems in category 'unused' is not analysed due to a compiler option being ignored'
In oxygen.1